qt/scientific_interfaces/ISISReflectometry/Common/multivisitors.hpp

```diff
@@ -0,0 +1,145 @@
+// clang-format off
+//  Boost.Varaint
+//  Multivisitors defined here
+//
+//  See http://www.boost.org for most recent version, including documentation.
+//
+//  Copyright Antony Polukhin, 2013.
+//
+//  Distributed under the Boost
+//  Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt).
+
+#ifndef BOOST_VARIANT_MULTIVISITORS_HPP
```

Comment:
This whole file looks copy/paste?

Reply:
I don't think this is used any more so I'll remove it